update-server-version.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+set -e
+
+if [[ "$(pwd)" != *"nats-docker" ]]; then
+	echo "$(basename ${0}) must be run from the repo top level"
+	exit 1
+fi
+
+current_version=$(ls -1 | sort | head -n 1)
+new_version="${1}"
+go_version="${2}"
+if [[ "${new_version}" == "" ]] || [[ "${go_version}" == "" ]]; then
+	echo "usage: ${0} <server version> <go version>"
+	echo "       ${0} 2.1.0 1.12.9"
+	exit 1
+fi
+
+echo "current version: ${current_version}"
+echo "new version: ${new_version}"
+echo "go version: ${go_version}"
+
+echo "updating files..."
```

Comment:
What about the windows nanoserver COPY instructions?
https://github.com/nats-io/nats-docker/blob/master/2.1.0/nanoserver1803/Dockerfile#L5
https://github.com/nats-io/nats-docker/blob/master/2.1.0/nanoserver1809/Dockerfile#L5

Reply:
That should work. The second `sed` takes care of that. Does it not work on macOS? :cold_sweat:

Reply:
Actually, just tried on macOS and sed does not work with given params:
```
current version: 2.1.0
new version: 2.1.0
go version: 1.12.9
updating files...
sed: illegal option -- -
usage: sed script [-Ealn] [-i extension] [file ...]
       sed [-Ealn] [-i extension] [-e script] ... [-f script_file] ... [file ...]
```

Reply:
But missed the second one that does for `nats:` so just need fixing for macOS and will try again :-)